trollsift/parser.py

```diff
@@ -45,8 +45,23 @@ def compose(self, keyvals):
         '''
         return compose(self.fmt, keyvals)
 
+    def partial_compose(self, keyvals):
+        """Convert parameters in `keyvals` to a string based on the *fmt* string.
+
+        This method is similar to compose, but accepts partial composing, i.e.,
+        not all parameters in `fmt` need to be specified in `keyvals`. Unspecified
+        parameters are left unchanged.
+
+        Args:
+            keyvals (dict): "Parameter --> parameter value" map
+
+        """
+        return partial_compose(self.fmt, keyvals)
```

Comment:
Should this be tested too? (just for coverage)

Reply:
This method doesn't exist anymore in my new commit. Whatever tests  the `compose` method should also be testing the equivalent of this now.